rewards/boost/boost_utils.py

```diff
@@ -46,8 +49,12 @@ def calc_boost_balances(
     native = Counter()
     non_native = Counter()
 
+    bvecvx_bals = sett_snapshot(chain, block, addresses.BVECVX, blacklist=True)
```

Comment:
It doesn't look like we're capturing bveCVX balances in the bveCVX/CVX LP here, just naked holdings of bveCVX?